DESCRIPTION

```diff
@@ -40,8 +40,8 @@ Suggests:
     rmarkdown,
     rpart,
     testthat
-Remotes: 
-    tidymodels/parsnip
+Remotes:
+    tidymodels/parsnip#631
 Config/Needs/website:
     tidymodels,
     tidymodels/parsnip,
```

Comment:
I think the pkgdown workflow fails because this does use the same branch as in `Remotes`